llama_index/storage/storage_context.py

```diff
@@ -51,12 +62,14 @@ def from_defaults(
             docstore (Optional[BaseDocumentStore]): document store
             index_store (Optional[BaseIndexStore]): index store
             vector_store (Optional[VectorStore]): vector store
+            graph_store (Optional[GraphStore]): graph store
 
         """
         if persist_dir is None:
             docstore = docstore or SimpleDocumentStore()
             index_store = index_store or SimpleIndexStore()
             vector_store = vector_store or SimpleVectorStore()
+            graph_store = graph_store or SimpleGraphStore()
```

Comment:
just checking: there should be minimal overhead in doing this right?

Otherwise, we should just not construct an object and leave it as None, so it doesn't impact other type of indices that don't use graph.

Reply:
It's the same as a vector_store essentially in terms of impact -> should be near-zero. 

Similar to how other indexes don't use the vector store, but we still instantiate it I guess